README.md

```diff
@@ -184,10 +229,15 @@ KEEN_WRITE_KEY=
 SENTRY_DSN=
 
 REDIS_UCLAPI_HOST=localhost
+
+STATIC_URL=http://localhost:8080/
```

Comment:
This should be `STATIC_URL=/static/` I think. We're not using webpack-serve anymore.